feat(changelog): key-manager-added-key-manager-events-in-audit-trail 2025-03-28 (#4731)
scaleway/docs-content: changelog/march2025/2025-03-28-key-manager-added-key-manager-events-in-audit-trail.mdx

---
title: Key Manager events in Audit Trail
status: added
date: 2025-03-28
category: security-identity
product: key-manager
---

### What is included

> You can now track encryption key creation, editing, rotation and deletion. Find out more about the Key manager supported Audit Trail endpoints in the [dedicated documentation](/audit-trail/reference-content/adt-supported-endpoints/#key-manager).

Reviewer: ```suggestion
You can now track encryption key creation, editing, rotation and deletion. Find out more about the Key Manager supported Audit Trail endpoints in the [dedicated documentation](/audit-trail/reference-content/adt-supported-endpoints/#key-manager).
```

> ### Key benefits:

Reviewer: ```suggestion
### Key benefits
```

> ### What's included:
> * Track encryption key creation, editing, rotation and deletion
> 
> ### Key benefits:
> * Full visibility within 2 seconds of action completion
> * Complete audit logs with user IDs, timestamps
> * Simple filtering by action type and principal
> * One-month log retention
> 
> Access this feature in the [Audit Trail section](https://console.scaleway.com/audit-trail/) of the Scaleway console.
> 

Reviewer: ```suggestion
### What is included

You can now track encryption key creation, editing, rotation and deletion. Find out more about the Key manager supported Audit Trail endpoints in the [dedicated documentation](/audit-trail/reference-content/adt-supported-endpoints/#key-manager).

### Key benefits:
- Full visibility within 2 seconds of action completion
- Complete audit logs with user IDs, timestamps
- Simple filtering by action type and principal
- One-month log retention

Access this feature in the [Audit Trail section](https://console.scaleway.com/audit-trail/) of the Scaleway console.
```
